GoogleForCreators/web-stories-wp
# Shopping: Add structured data

## Context



## Summary

Copy product data to post meta and render structured data on the front end. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create a story with a products. 
2. Save story. 
3. View front end. 
4. Look at markup and see structured data. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11153
